source/includes/administration/_authentication.md

```diff
@@ -59,8 +59,176 @@ Attributes | &nbsp;
-`logo`<br/>*string* | A base64 encoded data url for the logo to display on the login screen.
+`logo`<br/>*string* | A base64 encoded data url or url to an image for the logo to display on the login screen.
+`css`<br/>*string* | Custom css for the login button of the identity provider.
 `identityProviderUsers`<br/>*Array* | A list of objects containing the ids of users associated with the identity provider, and their subject ids.
 `connectionName`<br/>*string* | The connection name of the identity provider.
 `type`<br/>*string* | The type of authentication protocol. Possible values: OIDC, SAML.
 `parameters`<br/>*Array* | A list of parameters associated with the identity provider.
+
+<!-------------------- CREATE IDPS -------------------->
+
+#### Create identity provider
+
+`POST /identity_providers`
+
+Create a new identity provider
+
+```shell
+# Creates a new identity provider
+curl -X POST "https://cloudmc_endpoint/rest/identity_providers" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+> Request body example:
+
+```js
+{
+	"provider": "CUSTOM",
+	"type": "OIDC",
+	"connectionName": "CloudMC Google",
+	"displayName": "Google",
+	"parameters": [
+		{
+			"parameter": "issuerURL",
+			"value": "https://accounts.google.com"
+		},
+		{
+			"parameter": "clientId",
+			"value": "secret"
+		},
+		{
+			"parameter": "clientSecret",
+			"value": "secret"
+		}
+	]
+}
+```
+> The above command return JSON structured like this:
+
+```js
+{
+  "data": {
+    "provider": "CUSTOM",
+    "displayName": "Google",
+    "id": "da33bf85-6ba3-4214-a258-9442de149eff",
+    "connectionName": "CloudMC Google",
+    "type": "OIDC",
+    "parameters": [
+      {
+        "parameter": "issuerURL",
+        "id": "54d423fc-5ae2-4abc-bca6-9325376b6275",
+        "value": "https://accounts.google.com"
+      },
+      {
+        "parameter": "clientId",
+        "id": "a0538e7e-3049-45f0-97a5-83f55fc012e9",
+        "value": "secret"
+      },
+      {
+        "parameter": "clientSecret",
+        "id": "760c6c03-7c2f-4a18-a7c8-9ebc758ea33e",
+        "value": "secret"
+      }
+    ]
+  }
+}
+```
+
+Required | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the identity provider.
+`provider`<br/>*string* | The name of the provider. Possible values include the default providers (e.g GOOGLE), or CUSTOM for a custom user-defined provider.
+`displayName`<br/>*string* | The display name of the identity provider that will appear on the login screen.
+`connectionName`<br/>*string* | The connection name of the identity provider.
+`type`<br/>*string* | The type of authentication protocol. Possible values: OIDC, SAML.
+`parameters`<br/>*Array* | A list of parameters associated with the identity provider.
+
+Optional | &nbsp;
+---------- | -----------
+`logo`<br/>*string* | A base64 encoded data url or url to an image for the logo to display on the login screen.
```

Comment:
isn't it only optional if you have a non default provider? also, you should mention that default providers bring they're own values/

Reply:
https://github.com/cloudops/cloudmc-api-docs/pull/153/commits/91a783383ac46482d2bc8ba762081cb266376e64

Reply:
https://github.com/cloudops/cloudmc-api-docs/pull/153/commits/f1981cc0deadb19e62d0c45174765cb0e98b884c